packages/camera/camera/example/lib/main.dart

```diff
@@ -27,32 +27,33 @@ IconData getCameraLensIcon(CameraLensDirection direction) {
       return Icons.camera_front;
     case CameraLensDirection.external:
       return Icons.camera;
+    default:
+      throw ArgumentError('Unknown lens direction');
   }
-  throw ArgumentError('Unknown lens direction');
 }
 
-void logError(String code, String message) =>
+void logError(String code, String? message) =>
     print('Error: $code\nError Message: $message');
```

Comment:
If we have cases where there's no `message`, the format of this should probably conditionalize showing the second line. (Alternately, can we always pass a message?)